.github/workflows/main-ci.yml

```diff
@@ -88,6 +88,11 @@ jobs:
       { name: 'Install Python', uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c, with: { python-version: 3.12} },
       { name: 'Install Java', uses: actions/setup-java@99b8673ff64fbf99d8d325f52d9a5bdedb8483e9, with: { java-version: '17.0.5+8', distribution: 'temurin'} },
       { name: 'Setup dotnet', uses: actions/setup-dotnet@v4, with: {dotnet-version: '9.x', dotnet-quality: 'preview'}},
+      { name: 'Setup emsdk', uses: mymindstorm/setup-emsdk@v14, with: {version: '3.1.65'}},
+      { name: 'Verify emsdk',
+        run: 'emcc -v && export | grep EM'
+      },
+      { name: Install LLVM and Clang, uses: KyleMayes/install-llvm-action@v2, with: {version: "13.0"}},
```

Comment:
I think Clang is installed in the runners?

https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2204-Readme.md

Although that is Ubuntu 22.04 and not 20.04. And perhaps it is better to be explicit about which tools we need and don't assume they are present on the runner?

Reply:
Clang 13 is the latest for ubuntu 20.